src/Bicep.Cli/Commands/BuildParamsCommand.cs

```diff
@@ -105,5 +157,25 @@ public async Task<int> RunAsync(BuildParamsArguments args)
         private bool IsBicepFile(string inputPath) => PathHelper.HasBicepExtension(PathHelper.FilePathToFileUrl(inputPath));
 
         private bool IsBicepparamsFile(string inputPath) => PathHelper.HasBicepparamsExension(PathHelper.FilePathToFileUrl(inputPath));
+
+        private static readonly ImmutableHashSet<JTokenType> SupportedJsonTokenTypes = new[] { JTokenType.Object, JTokenType.Array, JTokenType.String, JTokenType.Integer, JTokenType.Float, JTokenType.Boolean, JTokenType.Null }.ToImmutableHashSet();
+
+        private static SyntaxBase ConvertJsonToBicepSyntax(JToken token) =>
+        token switch
+        {
+            JObject @object => SyntaxFactory.CreateObject(@object.Properties().Where(x => SupportedJsonTokenTypes.Contains(x.Value.Type)).Select(x => SyntaxFactory.CreateObjectProperty(x.Name, ConvertJsonToBicepSyntax(x.Value)))),
+            JArray @array => SyntaxFactory.CreateArray(@array.Where(x => SupportedJsonTokenTypes.Contains(x.Type)).Select(ConvertJsonToBicepSyntax)),
+            JValue value => value.Type switch
+            {
+                JTokenType.String => SyntaxFactory.CreateStringLiteral(value.ToString(CultureInfo.InvariantCulture)),
+                JTokenType.Integer => SyntaxFactory.CreatePositiveOrNegativeInteger(value.Value<long>()),
+                // Floats are currently not supported in Bicep, so fall back to the default behavior of "any"
+                JTokenType.Float => SyntaxFactory.CreateFunctionCall("json", SyntaxFactory.CreateStringLiteral(value.ToObject<double>().ToString(CultureInfo.InvariantCulture))),
+                JTokenType.Boolean => SyntaxFactory.CreateBooleanLiteral(value.ToObject<bool>()),
+                JTokenType.Null => SyntaxFactory.CreateFunctionCall("null"),
```

Comment:
>CreateFunctionCall [](http://example.com/codeflow?start=49&length=18)

Should we create a bicep null value instead of the `null()` function call?

Reply:
yes